i18n/en.json

```diff
@@ -3420,7 +3420,7 @@
   },
   {
     "id": "ent.message_export.global_relay.close_zip_file.app_error",
-    "translation": "Unable to close properly the zip file."
+    "translation": "Unable to properly close the zip file."
```

Comment:
Since we're in here, what if we just simplify this to `Unable to close the zip file.`

Reply:
Feel free to change, I was just correcting grammar.